json/flagd_definitions.go

```diff
@@ -2,8 +2,8 @@ package flagd_definitions
 
 import _ "embed"
 
-//go:embed flagd-definitions.json
-var FlagdDefinitions string
+//go:embed flags.json
+var FlagSchema string
 
 //go:embed targeting.json
-var Targeting string
+var TargetingSchema string
```

Comment:
We should track and change flagd references - https://github.com/open-feature/flagd/blob/main/core/pkg/evaluator/json.go#L407-L414

Reply:
Yes I am going to immediately absorb this in flagd when this is merged, since I want the better schema name.